src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -63,7 +69,7 @@ public void open() {
     }
 
     @Override
-    @SuppressWarnings({"checkstyle:CyclomaticComplexity", "checkstyle:NPathComplexity"})
+    @SuppressWarnings({"checkstyle:CyclomaticComplexity", "checkstyle:NPathComplexity", "checkstyle:MethodLength"})
```

Comment:
Surely the correct response to this warning is not to add a suppression but to refactor the method? And I don't think that refactoring is difficult: The body of each case statement would appear to be easily extracted as a method.

Reply:
Totally agree. I was just going for the fast way but I will make this change which was already in my mind.